data/modules/Debug/TestComponent.lua

```diff
@@ -0,0 +1,40 @@
+-- Copyright © 2008-2022 Pioneer Developers. See AUTHORS.txt for details
```

Comment:
is this file not completed?

Reply:
This file was committed both to provide an example of the API and also to act as a unittest to verify the system does in fact work. I've left it present but commented out as it's not required to run the test every time we start a new game.

Reply:
@Web-eWorks ok, why the file is not commented out completely, it seems that what is not commented out still does nothing?
Also, maybe it would be good to leave a mark like "uncomment for test", for code readers of my level.

Reply:
I'm happy with the way this file is, that only the code that makes it "live" is commented out.

Reply:
Leaving this file as is, as the uncommented portions are no-op and it's clear enough from the header comment why the file exists as far as I'm concerned.